### ISSUE TITLE
How to practice other models with kubeflow?

### ISSUE THREAD

**Issue by myqq0000** [kubeflow/kubeflow#891]
Hi there. I have tried to use kubeflow to train model.
But when I followed the guide to [submitting-a-tensorflow-training-job](https://github.com/kubeflow/kubeflow/blob/master/user_guide.md#submitting-a-tensorflow-training-job)
I can just train the TFCNN example.
There seems not mention how to change the model?
I can find this model with command `kubectl get -o yaml tfjobs ${CNN_JOB_NAME} `
And the model which will be trained can be seen in the returns:

> containers:
>         - args:
>           - python
>           **- tf_cnn_benchmarks.py**
>           - --batch_size=32
>           - --model=resnet50
>           - --variable_update=parameter_server
>           - --flush_stdout=true
>           - --num_gpus=2

Are there any ways to change it? 
Or maybe if I want to train other model I can't use Ksonnet?
Should I write my own job with [TFJob](https://github.com/kubeflow/tf-operator) ?

And by the way, where is the tf-benchmarks which is trained in this example stored?
I think maybe I need to package the model, the data set and tensorflow into an image and create a new job with this image?
Sorry about that I don't know kubernetes and docker very well. 

Thanks a lot.

**Comment by jlewi** [comment]
@myqq0000 Does that address your question? Can we close this issue

/label question
